archtitus.sh

```diff
@@ -19,15 +19,16 @@ echo -ne "
 -------------------------------------------------------------------------
                 Scripts are in directory named ArchTitus
 "
-    ( bash $SCRIPT_DIR/scripts/startup.sh )|& tee startup.log
-      source $CONFIGS_DIR/setup.conf
-    ( bash $SCRIPT_DIR/scripts/0-preinstall.sh )|& tee 0-preinstall.log
-    ( arch-chroot /mnt $HOME/ArchTitus/scripts/1-setup.sh )|& tee 1-setup.log
-    if [[ ! $DESKTOP_ENV == server ]]; then
-      ( arch-chroot /mnt /usr/bin/runuser -u $USERNAME -- /home/$USERNAME/ArchTitus/scripts/2-user.sh )|& tee 2-user.log
-    fi
-    ( arch-chroot /mnt $HOME/ArchTitus/scripts/3-post-setup.sh )|& tee 3-post-setup.log
-    cp -v *.log /mnt/home/$USERNAME
+mkdir logs
+( bash $SCRIPT_DIR/scripts/startup.sh )|& tee logs/startup.log
+source $CONFIGS_DIR/setup.conf
+( bash $SCRIPT_DIR/scripts/0-preinstall.sh )|& tee logs/0-preinstall.log
+( arch-chroot /mnt $HOME/ArchTitus/scripts/1-setup.sh )|& logs/tee 1-setup.log
+if [[ ! $DESKTOP_ENV == server ]]; then
+  ( arch-chroot /mnt /usr/bin/runuser -u $USERNAME -- /home/$USERNAME/ArchTitus/scripts/2-user.sh )|& tee logs/2-user.log
+fi
+( arch-chroot /mnt $HOME/ArchTitus/scripts/3-post-setup.sh )|& tee logs/3-post-setup.log
+cp -v *.log /mnt/home/$USERNAME
```

Comment:
This will need to be updated if you're moving where logs are stored. This is what copies the generated logs from installation to the user's home directory so they can be reviewed after rebooting.